student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
```

Comment:
- [ ] Please make sure that your class `Student` inherits from `Person` . Please check the following [link ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance.)to have more hints and implement it in order to make your app works as expected :+1: